Assets/MRTK/Core/Inspectors/ControllerPopupWindow.cs

```diff
@@ -195,9 +196,16 @@ public static void Show(MixedRealityControllerMapping controllerMapping, Seriali
 
             window = null;
 
+            if (mappedControllers == null)
+            {
+                mappedControllers = new List<string>();
```

Comment:
It feels slightly odd to me to allocate an empty list if this is null, instead of null checking in `RenderMappingList` and returning early if it's null.

Reply:
Though I guess this only gets called once when the button is clicked, it still feels odd to me to allocate an empty array that's never added to.